myapp/Dockerfile

```diff
@@ -12,4 +12,9 @@ RUN chmod +x /usr/bin/entrypoint.sh
 ENTRYPOINT ["entrypoint.sh"]
 EXPOSE 3000
 
+# node install
+RUN curl -SL https://deb.nodesource.com/setup_12.x | bash - \
```

Comment:
Node.js seems to be needed in order to run bootstrap.
without Node.js, it will cause this error below.

```
Could not find a JavaScript runtime. See https://github.com/rails/execjs for a list of available runtimes. (ExecJS::RuntimeUnavailable)
```